documentation/book/con-Define-Cluster-Roles.adoc

```diff
@@ -0,0 +1,41 @@
+[id='define-cluster-roles-{context}']
+= Defining `ClusterRoles`
+
+The Cluster Operator needs to operate using `ClusterRoles` that give it access to the necessary resources.
+Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the `ClusterRoles`.
+
+NOTE: Cluster administrator rights are only needed for the creation of the `ClusterRoles`.
+The Cluster Operator will not run under the cluster admin account.
```

Comment:
I've seen this as cluster-admin in the getting started docs.  Which is right @scholzj or is this a different thing?